versions.json

```diff
@@ -846,7 +846,17 @@
       "readme_url": "modules/masterfiles/ea81b7b790f1bf98cbf145d8700a06b37d74fb84.md",
       "readme_sha256": "5290c619aef65139e7378b75439b8668756fb91df1ee6a20d7627a00a24c2407",
       "steps": [
-        "run ./prepare.sh -y",
+        "run EXPLICIT_VERSION=3.18.5 EXPLICIT_RELEASE=1 ./prepare.sh -y",
+        "delete cfe_internal/core/watchdog/README.md",
+        "delete cfe_internal/enterprise/ha/ha_info.json",
+        "delete .github",
+        "delete inventory/README.md",
+        "delete lib/README.md",
```

Comment:
Actually, to me it would make some sense if the tarballs had the README, but the module did not. But that's the opposite of what is currently the case, and not a big deal.